Add simulation trace extension to file formats (#56)
UPPAALModelChecker/docs.uppaal.org: content/toolsAndAPI/file-formats.md

## Trace files

### Symbolic simulation traces

Symbolic simulation traces can be stored using the XTR format and files in this format have the file ending <tt>.xtr</tt>. The format was introduced in UPPAAL 3.0.

XTR files are intimately linked to the model from which they were generated. Any change (other than layout and white space changes) in the model may render the trace file unreadable. Even reordering some variable declarations will break the link.

Our parser library, libutap, contains a small utility called <tt>tracer</tt>, which can read XTR files and translate them to a human readable format. The library and the utility are released under the LGPL license and may thus be used as a starting point for writing custom trace analysis tools. More information about the library can be found on the UPPAAL web page.

### Concrete simulation traces

> Concrete simulation traces are stored in <tt>.uctr</tt> files, which are written in a JSON-like syntax, making them easy to parse with external editors.

Reviewer: Inconsistent wording between symbolic and concrete simulation trace description: `Symbolic simulation traces can be stored using ...` vs `Concrete simulation traces are stored in ...`

Author: Also fixed this consistency issue

> Concrete simulation traces are stored in <tt>.uctr</tt> files, which are written in a JSON-like syntax, making them easy to parse with external editors.

Reviewer: ```suggestion
Concrete simulation traces are stored in <tt>.uctr</tt> files, which are written in a JSON-like syntax, such that external editors can parse them.
```

Just to get rid of `easy`, as what is easy differs per person (i.e., what is easy for one, might not be easy for someone else). I prefer that documentation sticks to a more factual (technical) description.

Author: Yeah, fair point. It's my first time contributing to an open source project, plus I was in a bit of a hurry when re-reading, thank you for your comments :)
